src/track/trackmetadatataglib.cpp

```diff
@@ -967,6 +1032,20 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
         pTrackMetadata->setAlbumArtist(toQStringFirstNotEmpty(albumArtistFrame));
     }
 
+	QString rating;
+    const TagLib::ID3v2::FrameList popmRatingFrameList(tag.frameListMap()["POPM"]);
+	if (!popmRatingFrameList.isEmpty()) {
+		rating = QString::number(dynamic_cast<TagLib::ID3v2::PopularimeterFrame*>(popmRatingFrameList.front())->rating());
```

Comment:
This code duplicates part of the loop body. Initialize the rating with -1 and inside the loop body check if either of these conditions is met: (rating == -1) || (email == ...)